modules/gapi/test/common/gapi_imgproc_tests_common.hpp

```diff
@@ -14,6 +14,105 @@
 
 namespace opencv_test
 {
+// Draw random ellipses on given mat of given size and type
+static void initMatForFindingContours(cv::Mat& mat, const cv::Size& sz, const int type)
+{
+    cv::RNG& rng = theRNG();
```

Comment:
const if possible.

Reply:
You mean, for `cv::RNG&` ? Tried, didn't work